conan/api/subapi/remotes.py

```diff
@@ -31,6 +32,8 @@ def __init__(self, conan_api):
         self.conan_api = conan_api
         self._home_folder = conan_api.home_folder
         self._remotes_file = HomePaths(self._home_folder).remotes_path
+        # Wraps an http_requester to inject proxies, certs, etc
+        self._requester = ConanRequester(self.conan_api.config.global_conf, self.conan_api.cache_folder)
```

Comment:
We need to design, implement and document properly the "invalidating" and reloading of global.conf, like the requester will not get new conf after a ``config install``. I think this is perfectly good as default, but the "invalidation" should also be doable and documented. I'll create a ticket for this later.